pkgs/tools/admin/pulumi/get_latest_plugins.sh

```diff
@@ -0,0 +1,29 @@
+plugins=(
```

Comment:
```suggestion
#!/usr/bin/env bash

plugins=(
```

Reply:
Can you make the script executable?

Reply:
Added shebang and made the script executable. Thanks!